extensions/ql-vscode/src/remote-queries/view/RepositoriesSearch.tsx

```diff
@@ -10,20 +9,16 @@ interface RepositoriesSearchProps {
 
 const RepositoriesSearch = ({ filterValue, setFilterValue }: RepositoriesSearchProps) => {
   return <>
-    <TextInput
-      block
-      sx={{
-        backgroundColor: 'var(--vscode-editor-background);',
-        color: 'var(--vscode-editor-foreground);',
-        width: 'calc(100% - 14px)',
-      }}
-      leadingVisual={SearchIcon}
-      aria-label="Repository search"
+    <VSCodeTextField
+      placeholder='Filter by repository owner/name'
+      size='150'
```

Comment:
Another slight change from the old component: I wasn't able to set an equivalent to `width: 'calc(100% - 14px)'`. 

`size='100'` made the box slightly less than full-width, so I picked `150` as something slightly higher. The box still resizes appropriately when you resize the screen 🤷🏽

Reply:
I believe you can set any CSS property you want by setting `style` e.g.

```jsx
<VSCodeTextField style={{ width: 'calc(100% - 14px)' }} 
```

but if it looks ok then I don't mind which way we do it!